synthetic_data/distinct_generators/int_generator.py

```diff
@@ -20,4 +20,7 @@ def random_integers(
 
     :return: np array of integers
     """
+    if min_value == max_value:
+        max_value += 1
```

Comment:
standard fix for min_value == max_value

Reply:
if they are the same, though, why not just generate all the same values equal to len(num_rows)... in this scenario with `+=1`, you'll be generating data that is not equal to the original distribution

Reply:
i think it will generate values equal to the ones in the original
it is max index exclusive

Reply:
add test for this in `distinct_generators/test_int_generator.py`